SDKAndroid/src/test/java/app/cybrid/sdkandroid/tools/TestConstants.kt

```diff
@@ -166,4 +162,13 @@ object TestConstants {
             sellPrice = BigDecimal(2374000)
         )
     )
+
+    // -- JSON
+
+    val CREATE_CUSTOMER_SUCCESS = "{\"guid\":\"1234\",\"type\":\"individual\",\"created_at\":\"2022-06-23T07:08:16.718Z\",\"state\":\"storing\"}"
```

Comment:
Instead of writing out the JSON as strings like this. Can we write it out as JSON and call `toString` or `stringify` on it? It'll be easier to maintain and read.